packages/oidc-browser/src/refresh/refreshGrant.ts

```diff
@@ -122,7 +122,7 @@ export async function refresh(
     response,
     dpopKey !== undefined,
   );
-  const webId = await getWebidFromTokenPayload(
+  const { webId } = await getWebidFromTokenPayload(
```

Comment:
Is it necessary to retrieve the `clientId` and return it later (line 131) as well?